src/files/pyfingerprint/pyfingerprint.py

```diff
@@ -795,7 +795,7 @@ def storeTemplate(self, positionNumber = -1, charBufferNumber = 0x01):
                 for i in range(0, len(templateIndex)):
                     ## Index not used?
                     if ( templateIndex[i] == False ):
-                        positionNumber = i
+                        positionNumber = (256 * page) + i
```

Comment:
@anhquan223241 It would be nice if you can improve your pull request, so i can merge it :)